lib/elixir/lib/code.ex

```diff
@@ -1100,6 +1117,11 @@ defmodule Code do
     * `:columns` - when `true`, attach a `:column` key to the quoted
       metadata. Defaults to `false`.
 
+    * `:unescape` - when `false`, preserves escaped sequences. For example,
+      `"null byte\\t\\x00"` will be kept as is instead of being converted to a
+      bit string literal. This is useful in combination with
+      `quoted_to_algebra/2`. Defaults to `true`.
```

Comment:
```suggestion
    * `:unescape` (since v1.10.0) - when `false`, preserves escaped sequences.
      For example, `"null byte\\t\\x00"` will be kept as is instead of being converted
      to a bitstring literal. Note if you set this option to false, the resulting AST is no
      longer valid, but it can be useful to analyze/transform source code, typically in
      in combination with `quoted_to_algebra/2`. Defaults to `true`.
```